apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/RequestResponsePopulateMdcFilter.java

```diff
@@ -49,6 +51,17 @@ public class RequestResponsePopulateMdcFilter extends OncePerRequestFilter {
   /** Prefix for responses. */
   private static final String RESPONSE_PREFIX = "response";
 
+  /** obfuscated replacement for MBI. */
+  private static final String OBFUSCATED_MBI = "us-mbi|*********";
+
+  /** Regex Pattern to check for an /Patient request end-point. */
+  private static final Pattern PATIENT_REQUEST_REGEX = Pattern.compile("fhir/Patient");
+
+  /** Regex Pattern to check for an MBI. */
+  private static final Pattern MBI_REGEX =
+      Pattern.compile(
+          "us-mbi\\|[1-9][^SLOIBZsloibz0-9][^SLOIBZsloibz][0-9]-?[^SLOIBZsloibz0-9][^SLOIBZsloibz][0-9]-?[^SLOIBZsloibz0-9][^SLOIBZsloibz0-9][0-9][0-9]");
```

Comment:
As far as I know, BFD's synthetic beneficiaries follow an MBI pattern with `/Ss/` in the second position, like our so-called golden beneficiary: `1S00ABBAA00`. Do you intend on excluding synthetic beneficiaries with this pattern?

Reply:
I hadn't planned on it (unless you deem it necessary/useful). That 'filter' is obviously specific to a real MBI; I didn't see the necessity of filtering MBIs that weren't real.
As for testing this, I had to disable the HTTP POST assertion to even get access to a URL param...BUT, if it is a /Patient endpoint AND the us-mbi|<some real MBI> were to show up as a URL param, then it would be obfuscated.

Reply:
Just to be clear, I was suggesting that the pattern **does** exclude the synthetic identifiers. As you said, that's obvious from the pattern, but my question was whether or not this was the intentional, and I think it is. I suppose my question is somewhat reactionary to the *very* recent events that had some folks confused about our synthetic beneficiary format. I believe the only place we reference documentation about MBI formatting is in the context of RFC0020.

No change necessary. I'm just working through how we can better document these things for ourselves and our users. Thanks!